deploy-to-api.js

```diff
@@ -59,7 +59,7 @@ if (!_.isEmpty(apiToken)) {
 	};
 }
 
-const apiEndpointWithPrefix = url.resolve(apiEndpoint, '/v2/');
+const apiEndpointWithPrefix = url.resolve(apiEndpoint, '/v5/');
```

Comment:
I wonder, if on the longer term, this could be done with the SDK, using the SDK's pine request? Thus this bumping not needing to be done directly?

Reply:
Yeah that's a good point. I'll tackle that in a seperate PR though.